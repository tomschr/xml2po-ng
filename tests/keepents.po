msgid ""
msgstr ""
"Project-Id-Version: keep entities test\n"
"POT-Creation-Date: 2006-03-31 21:47+0200\n"
"PO-Revision-Date: 2006-03-31 21:48+0200\n"
"Last-Translator: Данило\n"
"Language-Team: српски\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: keepents.xml:6(title)
msgid "&language; is best language!"
msgstr "&language; је најбољши језик!"

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: keepents.xml:0(None)
msgid "translator-credits"
msgstr ""
