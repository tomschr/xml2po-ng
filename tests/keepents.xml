<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
"http://www.docbook.org/xml/4.4/docbookx.dtd" [
<!ENTITY % language "Serbian">
]>
<article>
<title>&language; is best language!</title>
</article>
