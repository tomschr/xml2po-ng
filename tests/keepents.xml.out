<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN" "http://www.docbook.org/xml/4.4/docbookx.dtd" [
<!ENTITY % language "Serbian">
]>
<article lang="keepents">
<title>&language; је најбољши језик!</title>
</article>
